stable/grafana/templates/deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
   {{- end }}
       containers:
         - name: {{ template "grafana.name" . }}
-          image: "{{ .Values.server.image }}"
+          image: "{{ default "grafana/grafana:4.6.3" .Values.server.image }}"
```

Comment:
The default image is already in the `values.yaml` file...

https://github.com/kubernetes/charts/blob/964cdd503f039b27f26628345e2364dc3c8c4cbe/stable/grafana/values.yaml#L7-L9

It doesn't need to be here as well.